### ISSUE TITLE
i cant log in to the dsf

### ISSUE THREAD

**Issue by tpycbl** [ReHamster/Alcatraz#26]
![image](https://user-images.githubusercontent.com/132508248/236485836-aee744ba-d831-4362-804a-f0d090ffabbb.png)
When you log in to the game, this error appears. Please help me how to fix it?

**Comment by pirate11944** [comment]
Of course, I didn’t, but when I tried to log into my Alcatraz account, I didn’t succeed (
I really want to play multiplayer(((
